### ISSUE TITLE
UX : Fix Accessibility Colors Contrast

### ISSUE THREAD

**Issue by fmontes** [dotCMS/core#25869]
### Problem Statement
The a11y colors contrast is not functioning properly.

### Steps to Reproduce
1. Go to the dotCMS page
2. Identify any sections with a11y colors contrast
3. Try to interact with the elements in the page

### Acceptance Criteria
- Check with UX team to get the proper contrast ratio
- Try and fix the accessibility as much as possible

### External Links
- [WAVE Chrome Extensiom](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)

<img width="678" alt="image" src="https://github.com/dotCMS/core/assets/751424/34840ea7-99b9-4836-919c-d96c0818e72b">

### Assumptions & Initiation Needs
- Assume the UX team has identified the correct contrast ratio

### Quality Assurance Notes & Workarounds
- If the contrast ratio cannot be fixed, provide a workaround for users to still access the content in the page.
